layouts/_default/single.html

```diff
@@ -26,11 +26,6 @@
 			{{ end }}
 			<h1>{{ .Title }}</h1>
 			<p class="lead">{{ .Params.lead | safeHTML }}</p>
-			{{ if ne .Params.toc false -}}
-			<nav class="d-xl-none" aria-label="Quaternary navigation">
-				{{ partial "sidebar/docs-toc.html" . }}
```

Comment:
This is unrelated to fixing search, but I saw it causing duplicate HTML ID errors when I was digging into the build errors.